docs/deployment/docker.md

```diff
@@ -108,10 +108,13 @@ root of the repo to speed up the build by reducing build context size:
 
 ```text
 .git
+.yarn/cache
+.yarn/install-state.gz
```

Comment:
These are specific to Yarn 3 and mentioned in the [migration docs](https://backstage.io/docs/tutorials/yarn-migration). I think it's best to skip them for now. Let's add the `*.local.yaml` ones tho!

Reply:
Thanks - I've updated the doc.

Reply:
Sorry! I've led you astray, I didn't realize we had already added the yarn patterns to create-app. Your original change was correct, let's revert to that and we're good to go 👍